src/apps/audit-trail/src/components/controls/index.js

```diff
@@ -1,19 +1,17 @@
 import { memo } from "react";
-import cx from "classnames";
+
+import Button from "@mui/material/Button";
+import ButtonGroup from "@mui/material/ButtonGroup";
 
 import { Search } from "@zesty-io/core/Search";
-import { Button } from "@zesty-io/core/Button";
-import { ButtonGroup } from "@zesty-io/core/ButtonGroup";
 
 import styles from "./styles.less";
 export default memo(function AuditControls(props) {
   return (
     <header className={styles.auditControls}>
-      <ButtonGroup className={styles.btnGroup}>
+      <ButtonGroup variant="contained">
```

Comment:
Zesty ButtonGroup is v different from Mui ButtonGroup. Does this maintain desired look and feel?

Reply:
I did decide to move these buttons from stand alone buttons to MUI ButtonGroup as it seemed appropriate since they are a group of related buttons.

Legacy

![Screen Shot 2022-05-09 at 1 03 08 PM](https://user-images.githubusercontent.com/22800749/167488485-8ab80930-0701-4dbd-bd6e-d993a160bd6b.png)


MUI ButtonGroup
![Screen Shot 2022-05-09 at 1 03 21 PM](https://user-images.githubusercontent.com/22800749/167488509-5a8efb00-0dbf-468b-b426-b78d8f72b4b6.png)

CC: @shrunyan  what is your feedback ? Is it okay to move these to Button Group or want to keep stand alone buttons?

Reply:
Excellent decision @d88naimi 
I see this as a change bringing the UI closer to the intent of the original component. With minimal thrash.